src/functions/durableOrchestration.ts

```diff
@@ -0,0 +1,36 @@
+import { app, HttpHandler, HttpRequest, HttpResponse, InvocationContext } from '@azure/functions';
+import * as df from 'durable-functions';
+import { ActivityHandler, OrchestrationHandler } from 'durable-functions';
+
+// Replace with the name of your Durable Functions Activity
+const activityName = 'hello';
+
+const orchestrator: OrchestrationHandler = function* (context) {
+    const outputs = [];
+    outputs.push(yield context.df.callActivity(activityName, 'Tokyo'));
+    outputs.push(yield context.df.callActivity(activityName, 'Seattle'));
+    outputs.push(yield context.df.callActivity(activityName, 'Cairo'));
+
+    return outputs;
+};
+df.orchestration('durableOrchestrator1', orchestrator);
+
+const helloActivity: ActivityHandler<string> = (_context: InvocationContext, input: string) => {
+    return `Hello, ${input}`;
+};
+df.activity<string>(activityName, { handler: helloActivity });
```

Comment:
Something that throws me off about this sample is that some Functions are named under the suffix `Handler` whereas others aren't.

For example, the Orchestrator and the Activity are called `orchestrator` and `helloActivity` respectively, but the http-triggered durable-client function is called `clientHandler`. And yet, they are "handlers" in terms of the new programming model.

Can we standardize the naming somehow? Do we want these to all have the suffix "handler", or do we want them to just have names that aren't informed by the programming model?

Reply:
That's fair. I don't have strong preferences on this, but ideally I'd like to pick whatever name feels the simplest for each use-case while being descriptive. Do you have a preference on how to name them?

Reply:
Done